docs/examples/try_me_out/availability_search.ipynb

```diff
@@ -94,7 +127,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.8.11 ('venv': venv)",
```

Comment:
why does this need to change? same for below like with interpreter and hash

Reply:
These must have been auto-generated as I didn't change these. Will see how to revert